third_party/move/move-compiler-v2/src/bytecode_generator.rs

```diff
@@ -1327,6 +1346,35 @@ impl<'env> Generator<'env> {
         }
     }
 
+    /// Generate a test for variants.
+    fn gen_test_variants_operation(
+        &mut self,
+        id: NodeId,
+        bool_temp: &mut Option<TempIndex>,
+        success_label: Label,
+        str: &QualifiedInstId<StructId>,
+        variants: Vec<Symbol>,
+        src: TempIndex,
+    ) {
+        let bool_temp = if let Some(t) = *bool_temp {
```

Comment:
```
    let bool_temp = *bool_temp.get_or_insert_with(||
       self.new_temp(Type::new_prim(PrimitiveType::Bool)));
```

Reply:
Ah, I was looking for that one, thanks. Done.